src/mono/mono/component/debugger-agent.c

```diff
@@ -2262,6 +2262,12 @@ mono_wasm_get_tls (void)
 	GET_TLS_DATA_FROM_THREAD (thread);
 	return tls;
 }
+
+bool
+mono_wasm_breakpoint_and_stepping_disabled (void)
```

Comment:
Should this be renamed to `mono_wasm_is_breakpoint_and_stepping_disabled`?

Reply:
done.